### ISSUE TITLE
Create node email

### ISSUE THREAD

**Issue by fraoustin** [gjthompson1/glue-public#12]
Now for send a email add a lambda node with code

> '''
> - Define a function called 'function' to be applied to the entire data frame
> - acccess the column names using df['column_name']
> e.g.
> 
> def function(df):
>   return df.describe().reset_index()
> '''
> import smtplib
> 
> def function(df):
>   me = 'my@email.com'
>   you = 'you@email.com'
>   smtp = 'smtp.gmail.com'
>   password = 'mypassword'
>   subject = "my test"
>   server = smtplib.SMTP_SSL(smtp)
>   server.login(me, password)
>   body = df.to_csv(sep=' ', index=False, header=False)
>   msg = f"Subject: {subject}\n\n{body}"
>   server.sendmail(
>       me,
>       you,
>       msg
>   )
>   server.quit()
>   return df

or after add email module in glue path

> '''
> - Define a function called 'function' to be applied to the entire data frame
> - acccess the column names using df['column_name']
> e.g.
> 
> def function(df):
>   return df.describe().reset_index()
> '''
> import smtplib
> from email.mime.multipart import MIMEMultipart
> from email.mime.text import MIMEText
> 
> def function(df):
>   me = 'my@email.com'
>   you = 'you@email.com'
>   password = 'password'
>   server = smtplib.SMTP_SSL('smtp.gmail.com')
>   server.login(me, password)
>   subject = f'test'
>   text = df.to_csv(sep=' ', index=False, header=False)
>   html = df.to_html()
>   msg = MIMEMultipart('alternative')
>   msg['Subject'] = subject
>   msg['From'] = me
>   msg['To'] = you
>   part1 = MIMEText(html, 'text')
>   msg.attach(part1)
>   part2 = MIMEText(html, 'html')
>   msg.attach(part2)
>   server.sendmail(
>       me,
>       you,
>       msg.as_string()
>   )
>   server.quit()
>   return df
> 

It's possible to add a email node ?

**Comment by gjthompson1** [comment]
Yes! I think this is a really good idea and is something that is on the road map.
